### ISSUE TITLE
Add Column for Shell Command Execution - Feature Request

### ISSUE THREAD

**Issue by Giulioski** [gazugafan/fingerpass#6]
Hello, nice work and thank you for all.
Could you add a column beside the Enter checkbox for a Shell Command (excludes Password+Enter) and a checkbox (Run Always As Admin) ?
Shell Comand Exe should be easy to be added.
Run as Admin for non admin users should receive the first time the privilege from an Admin User. However this could exposes a security issue if you don't pay attention how to implement this. For sure this is a second feature request that takes much more time to be implemented without any risks.

**Comment by gazugafan** [comment]
Sort of makes sense, but might be out of scope for what this is intended to do. Is something like https://github.com/resilar/HotFinger more what you're looking for?

**Comment by Giulioski** [comment]
> Sort of makes sense, but might be out of scope for what this is intended to do. Is something like https://github.com/resilar/HotFinger more what you're looking for?

Yes I know but Hotfinger is revealed as virus and developer is asking to add it to the Antivirus Exclusions. What I don't really want to do as long I don't know all the code and part of it is written in ASM

**Comment by gazugafan** [comment]
I can't imagine a use case for running another program based on what window is focused, but I guess there could be one. I'll keep this open, but it'll be really low priority since I'd never make use of it myself. If you'd like to tackle it, feel free to send a PR :)
